vizro-core/hatch.toml

```
@@ -37,7 +37,20 @@ cov-report = [
 ]
 example = "cd examples/{args:default}; python app.py"
 lint = "SKIP=gitleaks pre-commit run {args} --all-files"
+prep-release = [
+  "hatch version release",
+  "hatch run docs:scriv collect",
+  "echo 'Raise a PR to prepare main for release'"
+]
 pypath = "hatch run python -c 'import sys; print(sys.executable)'"
+release = [
+  "git tag -a v`hatch version` -m 'Release version'",
+  "git push origin v`hatch version`",
+  "hatch build",
+  "hatch publish {args}",
+  "hatch version patch,dev",
+  "echo 'Raise a PR to bump to the new development version'"
```

Is there a way we can add the version number in the string?

```suggestion
  "echo 'Raise a PR to bump to the new development version'"
```